docs/source/API/core/memory_spaces.md

```diff
@@ -35,6 +35,18 @@
 
 `Kokkos::HostSpace` is a [`MemorySpace` type](MemorySpaceConcept) representing traditional random access memory accessible from the CPU.  Except in rare instances, it should not be used directly, but instead should be used generically as an memory space.  For details, see [the documentation on the `MemorySpace` concept](MemorySpaceConcept).
 
+(SharedHostPinnedSpace)=
+## `Kokkos::SharedHostPinnedSpace`
+
+`Kokkos::SharedHostPinnedSpace` <sup>since 4.0</sup> is a [`MemorySpace` type](MemorySpaceConcept) alias which is accessible by all enabled [`ExecutionSpace` types](ExecutionSpaceConcept). The memory stays pinned on the host and is availabe on the device via zero copy access in small chunks (cachlines, memory pages, etc. depending on the backend). Writes to the memory in one `ExecutionSpace` become visible in other `ExecutionSpaces` at synchronization events. Which events trigger a synchronization depend on the backend specifics. Nevertheless, fences are synchronization events on all backends.
```

Comment:
```suggestion
`Kokkos::SharedHostPinnedSpace` <sup>since 4.0</sup> is a [`MemorySpace` type](MemorySpaceConcept) alias which is accessible by all enabled [`ExecutionSpace` types](ExecutionSpaceConcept). The memory stays pinned on the host and is availabe on the device via zero copy access in small chunks (cache lines, memory pages, etc. depending on the backend). Writes to the memory in one `ExecutionSpace` become visible in other `ExecutionSpaces` at synchronization events. Which events trigger a synchronization depend on the backend specifics. Nevertheless, fences are synchronization events on all backends.
```
consider breaking up in multiple lines